server/src/main/java/org/opensearch/index/store/remote/fc/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * This package contains class used for File System Caching for remote store
+ */
+package org.opensearch.index.store.remote.fc;
```

Comment:
the `fc` package name is obscure.. please spell this out as `org.opensearch.index.store.remote.filecache`